src/unix/process.c

```diff
@@ -498,7 +498,7 @@ int uv_spawn(uv_loop_t* loop,
   } else
     abort();
 
-  uv__close(signal_pipe[0]);
+  close(signal_pipe[0]);
```

Comment:
Can I suggest splitting `uv__close()` into `uv__close()` and `uv__close_nocheckstdio()` (name TBD) and calling the latter here?  `uv__close()`, besides checking that `fd > STDERR_FILENO`, also preserves `errno` and checks for `EINTR`/`EINPROGRESS`.  Calling `uv__close_nocheckstdio()` also makes it self-descriptive why we're not calling `uv__close()` here.